bophono/data/ends.csv

```diff
@@ -31,31 +31,31 @@
 ིམས,im~,im,im,im
 ིལ,il,il,il,il
 ིའི,i:,i:,i,i
-ིའིའོ,i:|-o,,i'o,io
-ིའོ,i|-o,,i'o,io
-ིའང,i|-ang,,i'ang,iang
-ིའམ,i|-am,,i'am,iam
+ིའིའོ,i:|-o,,i’o,io
+ིའོ,i|-o,,i’o,io
+ིའང,i|-ang,,i’ang,iang
+ིའམ,i|-am,,i’am,iam
 ིར,ir,ir,ir,ir
 ིས,i',i,i,i
 ུ,u,u,u,u
 ུག,uk,ɨx,uk,uk
 ུགས,uk,ɨx,uk,uk
 ུང,ung,ong,ung,ung
 ུངས,ung~,ong,ung,ung
-ུད,ü',ut,ü,ü
```

Comment:
for `u` vs `ü` see https://github.com/Esukhia/bophono/issues/22#issuecomment-2345830346 , it should be `ü`

Reply:
Alright then, forgot to read the addendum :)